extensions/map-layers-formats/src/ArcGisFeature/ArcGisFeatureProvider.ts

```diff
@@ -215,11 +215,11 @@ export class ArcGisFeatureProvider extends ArcGISImageryProvider {
     // Note: needs to be checked on the layer metadata, service metadata advertises a different set of formats
     //       Also, since PBF format does not support floating points, there is no point using this format if supportsCoordinatesQuantization is not available.
     if (this._layerMetadata.supportedQueryFormats) {
-      const formats: string[] = this._layerMetadata.supportedQueryFormats.split(", ");
-      if (formats.includes("PBF") && this._supportsCoordinatesQuantization) {
-        this._format = "PBF";
-      } else if (formats.includes("JSON")) {
-        this._format = "JSON";
+      const formats: string[] = this._layerMetadata.supportedQueryFormats.split(",").map((s: string) => s.trim().toLowerCase());
```

Comment:
why not trim before split?

Reply:
This way trims each comma-separated component. Trimming first wouldn't do that.